app/navigation/Navigator.scala

```diff
@@ -47,6 +49,13 @@ class Navigator @Inject() () {
     _ => routes.CheckYourAnswersController.onPageLoad
   }
 
+  def complexRoute(answers: UserAnswers): Call =
```

Comment:
This is very generic name. Can we rename it - distributionsNextRoute

Reply:
done